src/routes/user/+page.svelte

```diff
@@ -1,19 +1,84 @@
 <script>
     // Framework imports
+    import { getToastStore } from "@skeletonlabs/skeleton";
     import { goto } from "$app/navigation";
     import { onMount } from "svelte";
 
-    // Functions
-    function fetchUserID() {
-        // ToDo fetch user id from backend
-        return 1;
-    }
+    // Import 🐓
+    import { ExponentialBackoff, handleAll, retry } from "cockatiel";
+
+    // Util library import for url routes
+    import { urls } from "$lib/util.js";
 
-    function redirectToUserpage(id) {
-        goto(`/user/${id}`);
-    }
+    // Component imports
+    import UserPageLoadingPlaceholders from "./userPageLoadingPlaceholders.svelte";
+    import LoggedInUser from "./loggedInUser.svelte";
 
-    onMount(() => {
-        redirectToUserpage(fetchUserID());
+    // Definitions
+    // Policy for restarting backend fetched up to 5 times if there's no reply
+    const retryPolicy = retry(handleAll, {
+        maxAttempts: 5,
+        backoff: new ExponentialBackoff(),
+    });
+
+    // Get Toaststore
+    const toastStore = getToastStore();
+
+    let loading = true;
+    let error = null;
+    let personalData = null;
+
+    // Toast settings
+    const t = {
```

Comment:
So ist eig. schöner, auch wenn Skeleton das anders macht.
```suggestion
    const toastSettings = {
```